.github/build/friendly-filenames.json

```diff
@@ -0,0 +1,33 @@
+{
+    "android-arm64": { "friendlyName": "android-arm64-v8a" },
+    "darwin-amd64": { "friendlyName": "macos-64" },
```

Comment:
this should be `darwin-amd64` in order to not change the format from previous release